src/Uno.CrossTargetting.props

```diff
@@ -6,9 +6,6 @@
 		<Product>$(AssemblyName) ($(TargetFramework))</Product>
 		<DefaultLanguage>en-US</DefaultLanguage>
 
-		<!-- Version management is now done through NuGet, this avoids issues related version mismatch -->
-		<Version>255.255.255.255</Version>
-
```

Comment:
Already exists in https://github.com/unoplatform/uno/blob/55c0447b1c5367032fa81695f8c2a2684292e6cc/src/Directory.Build.props#L26-L27

so this seems redundant here.